service/core/action/author/list.go

```diff
@@ -68,13 +64,8 @@ func list(w http.ResponseWriter, r *http.Request) {
 
 	defer resp.Body.Close()
 
-	usersWithPermission := []permission{}
-	json.NewDecoder(resp.Body).Decode(&usersWithPermission)
 	users := []model.Author{}
```

Comment:
Taking this issue into consideration slice returns null value instead of empty slice so where ever slice is used then 
declare it with **make(type, length)** users = make([]model.Author, 0)